css/styles.css

```diff
@@ -0,0 +1,337 @@
+body {
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    background-color: #0D0D11;
```

Comment:
Set common font-size, line-height, color for text on all pages. 
It will default styles for text without classes.